application.go

```diff
@@ -34,6 +34,25 @@ type Applications struct {
 	Apps []Application `json:"apps"`
 }
 
+// IpAddress is used by Ip Per Task Funcionality https://mesosphere.github.io/marathon/docs/ip-per-task.html
```

Comment:
Please call the struct `IPAddress` to satisfy [Golang's initialism rule](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).

Style improvements and typo fixes:

_Ip Per Task Funcionality_ ==> _IP-per-task functionality_